portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,17 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div class="bg"></div>
```

Comment:
Since this class is encompassing all of the page's content.. suggest naming it something more apt, instead of "bg" -- or don't make a separate class and just stylize the body itself.

Reply:
Renamed to background